features/gpc.yml

```diff
@@ -0,0 +1,7 @@
+name: Global privacy control
+description: The `Sec-GPC` request header indicates whether the user consents to a website or service selling or sharing their personal information with third parties.
```

Comment:
This feature is new to me, but—skimming the spec a bit—it looks like sending the header at all (https://w3c.github.io/gpc/#the-sec-gpc-header-field-for-http-requests and https://w3c.github.io/gpc/#extensibility-of-the-sec-gpc-field-value) is the indicator, not the value:

```suggestion
description: The `Sec-GPC` request header indicates that the user does not consent to a website or service selling or sharing their personal information with third parties.
```